docs/de/user/configuration.rst

```diff
@@ -66,7 +66,9 @@ Zwei-Faktor-Authentifizierung (2FA)
     Die Zwei-Faktor-Authentifizierung (2FA) dient dem Identitätsnachweis eines Nutzers mittels der
     Kombination zweier verschiedener und insbesondere unabhängiger Komponenten (Faktoren).
 
-https://de.wikipedia.org/wiki/Zwei-Faktor-Authentifizierung
+    https://de.wikipedia.org/wiki/Zwei-Faktor-Authentifizierung
+    
+**Warning**: enabling 2FA from the configuration interface is only possible is it has been authorized before in `app/config/parameters.yml` by setting the *twofactor_auth* property to `true` (do not forger to empty `/var/cache` after modification).
```

Comment:
I am a bit lazy to clone the repo for this small translation, so I post it below:
```
**Warnung:** Das Aktivieren von 2FA über das Konfigurations-Interface ist nur möglich, wenn vorher in der `app/config/parameters.yml` die `twofactor_auth`-Eigenschaft auf `true` gesetzt wurde (nach der Konfiguration das Leeren des Cache unter `/var/cache` nicht vergessen).
```